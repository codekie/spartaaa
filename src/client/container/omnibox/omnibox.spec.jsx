import React from 'react';
import { shallow } from 'enzyme';
import ConnectedOmnibox from './omnibox';

it('should render correctly', () => {
    expect(shallow(<ConnectedOmnibox />).debug()).toMatchSnapshot();
});
