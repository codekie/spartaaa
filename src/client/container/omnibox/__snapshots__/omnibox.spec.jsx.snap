// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
"<ContextConsumer>
  [function bound renderWrappedComponent]
</ContextConsumer>"
`;
